core/modules/replica/EchoRequest.cc

```diff
@@ -83,7 +83,7 @@ EchoRequest::EchoRequest(ServiceProvider::Ptr const& serviceProvider,
                          shared_ptr<Messenger> const& messenger)
     :   RequestMessenger(serviceProvider,
                          io_service,
-                         "REPLICA_ECHO",
+                         "ECHO",
```

Comment:
didn't you make this TEST_ECHO in WorkerEchoRequest? Do you want to be consistent?

Reply:
You're right!